### PR TITLE
feat: added helius transaction types

### PR DESCRIPTION
**Pull request by qudo-code**
# Additional Transaction Types
- Bringing in some additional transaction types from the monorepo.
- Ticked up package version to `1.0.0` due to the breaking change with how types are now imported. 
- Fixed documentation still referring to old type imports.

### CODE REVIEW

File: src/Helius.ts [hunk 1]
```diff
@@ -156,8 +156,8 @@ export class Helius {
             const webhook = await this.getWebhookByID(webhookID);
             const accountAddresses = webhook.accountAddresses.concat(newAccountAddresses)
             webhook.accountAddresses = accountAddresses;
-            if (accountAddresses.length > 10000) {
-                throw new Error(`a single webhook cannot contain more than 10,000 addresses`)
+            if (accountAddresses.length > 100000) {
```
Comment (NicolasPennie) [inline src/Helius.ts:159]: nit: I always like writing intervals of 1000 like this: `100_000`. makes it easier to read.

File: src/types/enums.ts [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+    TransactionType.NFT_AUCTION_UPDATED,
+    TransactionType.NFT_AUCTION_CANCELLED,
+    TransactionType.NFT_PARTICIPATION_REWARD,
+    TransactionType.BURN_NFT,
+    TransactionType.NFT_RENT_LISTING,
+    TransactionType.NFT_RENT_CANCEL_LISTING,
+    TransactionType.NFT_RENT_UPDATE_LISTING,
+    TransactionType.NFT_RENT_ACTIVATE,
+    TransactionType.NFT_RENT_END,
+    TransactionType.ATTACH_METADATA,
```
Comment (NicolasPennie) [inline src/types/enums.ts:170]: I think this is out of date. Its missing the PNFT migration

File: src/types/types.ts [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    innerInstructions: InnerInstruction[];
+};
+
+export type InnerInstruction = {
+    accounts: string[];
+    data: string;
+    programId: string;
+}
+
+export interface EnrichedTransactionResponseV2 {
```
Comment (NicolasPennie) [inline src/types/types.ts:115]: We can probably name this something more user friendly. Ideally the names match the OpenApi spec.

Comment (mertimus) [inline src/types/types.ts:115]: we probably shouldn't call this V2 in this context

File: README.md [hunk 4]
```diff
@@ -179,11 +179,15 @@ To read more about the most powerful NFT API on Solana, see [our docs](https://d
 To get all the tokens for an NFT collection:
 
 ```ts
-import { Helius, Types } from "helius-sdk";
+import {
+  type Collections
```
Comment (mertimus) [inline README.md:183]: maybe I'm wrong on this but I think the README examples should be in JS vs. TS

File: src/types/types.ts [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    feePayer: string;
+    signature: string;
+    slot: number;
+    timestamp: number;
+    nativeTransfers: NativeTransfer[] | null;
+    tokenTransfers: TokenTransfer[] | null;
+    accountData: AccountData[];
+    transactionError: TransactionError | null;
+    instructions: Instruction[];
+    events: [];
```
Comment (NicolasPennie) [inline src/types/types.ts:120]: This is an object and can either be a swap event or a nft event

**Review by NicolasPennie — APPROVED** [review]

**Pull request merged**